lib/tasks/reindex.rake

```diff
@@ -28,3 +28,10 @@ namespace :geoblacklight do
     GeoResourceReindexer.reindex_geoblacklight
   end
 end
+
+namespace :orangelight do
+  desc "Issue rabbitmq messages that each complete orangelight resource is updated"
+  task reindex: :environment do
+    OrangelightReindexer.reindex_orangelight
```

Comment:
Maybe you should force `ENV["BULK"] = true` here if it's important, and test for it in the message.

Reply:
I like this idea

Reply:
@tpendragon I forced it in the service. Is 1b10d38 what you were thinking?

Reply:
Looks good to me.